foundations/installations/installations.md

```diff
@@ -273,6 +269,73 @@ Once you have successfully met both of these requirements, you should be able to
 
 </details>
 
+<details markdown="block">
+<summary class="dropDown-header">WSL2</summary>
+
+Using WSL2 is an quick and easy way to get started with using Linux, allowing you to run a Linux distribution from within Windows. WSL2 is available on Windows 10 version 2004 and higher (Build 19041 and higher) and Windows 11.
+
+<div class="lesson-note" markdown="1">
+<h4>WSL2 and Linux instructions</h4>
+Because WSL2 is a full-fledged Linux distribution, almost everything that the curriculum teaches about Linux is also applicable to WSL2. In future lessons, whenever there are instructions that differ by OS, you should follow the Linux instructions, unless the lesson includes WSL2-specific instructions.
+</div>
+
+### Step 1: Installations
+
+#### Step 1.1: WSL2 and Virtual Machine Platform
+
+In order to make WSL2 work we'll first need to ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are enabled on your system.
+
+- Search for Control Panel in your applications, and open it
+- Select **Programs**
+- Select **Turn Windows Features on or off** (under the Programs and Features header)
+- Ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are toggled on, and if not toggle them on.
+- When prompted, restart your computer
+
+#### Step 1.2: Installing WSL2
+
+- Open PowerShell in administrator mode by searching for it in your applications, right clicking the top option, and then selecting run as administrator. You might get a prompt asking if you want to allow Windows Powershell to make changes to your device: click yes.
+- Enter the following command
+
+  ~~~powershell
+  wsl --install
+  ~~~
```

Comment:
Very small change, can you convert this codeblock over to using backtick fencing @Asartea ?

I only ask because we're already past this lesson in the process of updating the style, and I don't want the legacy style to sneak in.

Reply:
✅ Done